src/main/resources/db/changelog/changes/07-insert-data-to-users_roles-table.yaml

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-data-to-users_roles-table
+      author: Bohdan
+      changes:
+        - insert:
+            tableName: users_roles
+            columns:
+              - column: {name: role_id, value: "1"}
+              - column: {name: user_id, value: "2"}
+        - insert:
+            tableName: users_roles
+            columns:
+              - column: { name: role_id, value: "2"}
+              - column: { name: user_id, value: "1"}
```

Comment:
use this to avoid hardcoded values:
```tableName: users_roles
            columns:
              - column:
                  name: user_id
                  valueComputed: (SELECT id FROM users WHERE email='admin@admin.admin')
              - column:
                  name: role_id
                  valueComputed: (SELECT id FROM roles WHERE name='ROLE_ADMIN')```